src/hotspot/share/runtime/os.hpp

```diff
@@ -668,6 +668,13 @@ class os: AllStatic {
   static void flockfile(FILE* fp);
   static void funlockfile(FILE* fp);
 
+  // A safe implementation of realpath which will not cause a buffer overflow if the resolved path
+  //   is longer than PATH_MAX.
```

Comment:
Nit: remove leading space to align with text on previous line.

Reply:
done.